app/src/main/java/com/android/sample/ui/recipeOverview/RecipeOverview.kt

```diff
@@ -267,26 +289,30 @@ private fun RecipeDescription(currentRecipe: Recipe) {
     Spacer(modifier = Modifier.size(SMALL_PADDING.dp))
     currentRecipe.category?.let { Tag(it) }
     Spacer(modifier = Modifier.size(SMALL_PADDING.dp))
-    Row(
-        horizontalArrangement = Arrangement.Start,
-        verticalAlignment = Alignment.CenterVertically,
-        modifier = Modifier.testTag(RATING_ICON)) {
-          // Display of the rating Icon
-          Icon(
-              painter = painterResource(R.drawable.star_rate),
-              contentDescription = stringResource(R.string.star_rate_description),
-              modifier = Modifier.testTag(RECIPE_STAR).size(OVERVIEW_RECIPE_STAR_SIZE.dp),
-              tint = starColor)
-
-          Spacer(modifier = Modifier.size(SMALL_PADDING.dp))
-
-          // Rating Text
-          Text(
-              text = RATE_VALUE,
-              modifier = Modifier.testTag(RECIPE_RATE),
-              style = MaterialTheme.typography.bodyMedium,
-              color = MaterialTheme.colorScheme.onSecondary)
-        }
+
+    if (fridgeIngredientsCount > 0) {
+      Row(
+          horizontalArrangement = Arrangement.Start,
+          verticalAlignment = Alignment.CenterVertically,
+          modifier = Modifier.testTag(RECIPE_FRIDGE_INGREDIENTS)) {
+            // Display of the rating Icon
+            Icon(
+                painter = painterResource(R.drawable.fridgeicon),
+                contentDescription = stringResource(R.string.fridge_ingredients_description),
+                modifier = Modifier.size(OVERVIEW_RECIPE_STAR_SIZE.dp),
+                tint = graySlate)
+
+            Spacer(modifier = Modifier.size(SMALL_PADDING.dp))
+
+            // Rating Text
```

Comment:
Great work on this file! 🎉 Just a small thing I noticed—there’s an old comment lingering around that seems to have outlived its purpose. It’s like a sticky note that got forgotten on the fridge. 😄 Clearing it out will keep the codebase clean and focused. Awesome job otherwise! 🚀